openml/flows/flow.py

```diff
@@ -1,80 +1,358 @@
 from collections import OrderedDict
+import re
+
+import six
 import xmltodict
-import sklearn
 
 from .._api_calls import _perform_api_call
-from .functions import _check_flow_exists
+from openml.util import oml_cusual_string
 
 
 class OpenMLFlow(object):
     """OpenML Flow. Stores machine learning models.
 
+    Flows should not be generated manually, but by the function
```

Comment:
In order to maintain a high quality of flows on the server, I think the upload should be as restrictive as positive. Having the creation of a flow handled by a helper class/function takes away a lot of possibilities to to upload faulty flows to the server. I will update the class.